### PR TITLE
Feat: Add Note Command, Add Note UI & Add Note Date Saving & Shortcut Keys for Note

### LINKED ISSUE #37: Create a

### LINKED ISSUE #38: Implement date saving when you add notes

### PR DESCRIPTION
**Pull request by elroygohjy**
**Changes** Update from #90 

- Offset for every additional note window opened.
- Changed all shortcut keys for note to ALT + Q, W, T, S
- NoteDate to not show for new notes (pending further changes after discussion)
- Amended ALT + T command to add current DateTime on current text cursor, and to set the new text cursor to end of new DateTime for easier line breaks. 
- Refactor for [pull #88](https://github.com/AY2122S1-CS2103T-W08-1/tp/pull/88)

**Todos**
- I forget to refactor to person index note/ (I will do this when Elton is done with group command, and I am doing Group note for 1.2b)

### CODE REVIEW

**Review by kaixin-hc — COMMENTED** [review]
thanks for the quick refactor + the windows!

**some things from testing on mac**
Screenshot 2021-10-11 at 11 51 47 PM
:

Save is unusable bc it prints ß ... 

Do you mind changing the hotkey to control? I think it will help, and also solve the † which shows up when you drop the timestamp.

**Other comments (no action needed rn)**
- Date is still visible in the addressbook list.
- The confirmation message displays Last Edited, which I think is fine and pretty nice, actually
- i just realised that there's no line wrapping in txt files ahahahaha rip to them 
- The confirmation message isn't that useful -- it only says that you opened note, not whether or not you edited the note or closed without saving. In this sense it feels like it is lagging.

**TLDR:** Can you change the mac commands from ALT - key to CTRL -key?

**Review by EltonGohJH — APPROVED** [review]
LGTM
